package.json

```diff
@@ -13,7 +13,8 @@
     },
     "./dist/style.css": {
       "require": "./dist/style.css",
-      "import": "./dist/style.css"
+      "import": "./dist/style.css",
+      "style": "./dist/style.css"
```

Comment:
Wouldn't it make sense then to set it in the `.` import? I think that would make `@import "@vector-im/compound-web"` work?

Reply:
Oh good idea!